pkg/iac/scanners/terraformplan/tfjson/test/testdata/plan_with_template.json

```diff
@@ -0,0 +1,480 @@
+{
```

Comment:
This file can be minified

Reply:
Yes but it's nice to be able to read the test case input if any debugging is needed.

Reply:
I agree